### ISSUE TITLE
MacOS problem with fetch-params.sh

### ISSUE THREAD

**Issue by edubai** [zcash/zcash#6367]
I made some changes to fetch-params.sh file to make it work with MacOS since it keeps giving me *line 8: conditional binary operator expected* error.

here is the fixed code:

```shell
#!/usr/bin/env bash

export LC_ALL=C
set -eu

SCRIPT_NAME=$(basename $0)

if [ -z "${XDG_CACHE_HOME+x}" ]; then
    XDG_CACHE_HOME="${HOME}/.cache"
fi

if [ -z "${XDG_RUNTIME_DIR+x}" ]; then XDG_RUNTIME_DIR="${XDG_CACHE_HOME}"; fi



uname_S=$(uname -s 2>/dev/null || echo not)

if [ "$uname_S" == "Darwin" ]; then
    PARAMS_DIR="$HOME/Library/Application Support/ZcashParams"
else
    PARAMS_DIR="$HOME/.zcash-params"
fi
```

**Comment by sellout** [comment]
@edubai Thanks! Sorry for breaking this. I made some changes to your version. Could you check the version in the PR (#6376) and let me know if it works for you?

**Comment by daira** [comment]
Please re-open if the fix in #6367 doesn't work for you.
